pkg/ticker/ticker_test.go

```diff
@@ -12,6 +12,23 @@ import (
 	"github.com/openservicemesh/osm/pkg/kubernetes/events"
 )
 
+func TestInitTicker(t *testing.T) {
+	assert := assert.New(t)
+	mockConfigurator := configurator.NewMockConfigurator(gomock.NewController(t))
+	mockConfigurator.EXPECT().GetConfigResyncInterval().Return(time.Duration(0))
+
+	events.GetPubSubInstance().Subscribe(announcements.TickerStart)
+	events.GetPubSubInstance().Subscribe(announcements.TickerStop)
+
+	assert.Nil(rTicker)
+	ticker := InitTicker(mockConfigurator)
+	assert.NotNil(ticker)
+	assert.NotNil(rTicker)
+
+	newTicker := InitTicker(mockConfigurator)
+	assert.Equal(ticker, newTicker)
```

Comment:
`assert.Same()` would be a slightly tighter check here, though that might be getting too into implementation details.

https://pkg.go.dev/github.com/stretchr/testify/assert#Same